README.md

```diff
@@ -150,3 +150,30 @@ When running the app locally, ensure you are using the development `GOOGLE_SHEET
 (See google drive for this)
 
 If updating your environment variable doesn't work, try `rm -rf _build` and then starting your server again
+
+### Image uploads
+
+##### File size
+
+The Wagtail cms allows for a maximum file size of 10MB.
+
+Althouth I would suggest that files be no bigger than 1MB.
+
+Be aware though that many uploads of high resolution images will result in increased costs from our file hosting provider.
+
+##### Resolution
+
+Be aware that if an image has too high a resolution, clients will have to download more onto their device causing pages to take longer to load.
+
+If an image has a too low resolution, then the image could be pixilated on high resolution displays.
```

Comment:
I think you need to provide a solution here. 
> If possible, please test your proposed image on the site's staging area so that you can determine how this will look